yolox/evaluators/coco_evaluator.py

```diff
@@ -27,9 +27,33 @@
 )
 
 
+def per_class_AR_table(coco_eval, class_names=COCO_CLASSES, headers=["class", "AR"], colums=6):
+    per_class_AR = {}
+    recalls = coco_eval.eval["recall"]
+    # dimension of recalls: [TxKxAxM]
+    # precision has dims (iou, cls, area range, max dets)
```

Comment:
precision -> recall?

Reply:
precision points to AP; recall points to AR。
Precision is obtained using coco_eval.eval["precision"]; 
recall is obtained using coco_eval.eval["recall"]. 
The parameters for obtaining the results are defined in the class Params under pycocotools/cocoeval.py

Reply:
```suggestion
    # recall has dims (iou, cls, area range, max dets)
```
is what I mean.

Reply:
Modified, thank you